src/mono/cmake/options.cmake

```diff
@@ -50,11 +54,12 @@ option (DISABLE_CRASH_REPORTING "Disable crash reporting subsystem")
 option (DISABLE_ICALL_TABLES "Enable separate icall table library")
 option (ENABLE_ICALL_EXPORT "Export icall functions")
 option (ENABLE_ICALL_SYMBOL_MAP "Generate tables which map icall functions to their C symbols")
-option (ENABLE_MINIMAL "Set many DISABLE_ options at once. Comma-separated list of lowercase DISABLE_ options ie. 'jit,simd' etc.")
 option (ENABLE_PERFTRACING "Enables support for eventpipe library")
 option (ENABLE_INTERP_LIB "Enable separate interpreter library")
 option (ENABLE_LAZY_GC_THREAD_CREATION "Enable lazy runtime thread creation, embedding host must do it explicitly")
 
+set (GC_SUSPEND "default" CACHE STRING "GC suspend method (default, coop, hybrid)")
```

Comment:
Shouldn't we have an explicit option for preemptive, given that it won't be the default on all platforms?